### PR TITLE
Three new Opengl settings for Windows [GUI]

### PR DESCRIPTION
**Pull request by ssakash**

### CODE REVIEW

**Comment by ssakash** [comment]
@gregory38 
I needed them to be defined at resource.h to use them. or else, it will give a undeclared identifier error. take a look at the IDC's which are defined at both of those place since if it wasn't, it will cause compilation errors.

**Comment by gregory38** [comment]
I said to use same name/identifier as linux. Of course you still need to define them. Or change the linux name.

**Comment by ssakash** [comment]
I misunderstood it then :p , it's ready to be merged.

File: plugins/GSdx/GSSettingsDlg.cpp [hunk 1]
```diff
@@ -194,6 +194,9 @@ void GSSettingsDlg::OnInit()
 	CheckDlgButton(m_hWnd, IDC_AA1, theApp.GetConfig("aa1", 0));
 	CheckDlgButton(m_hWnd, IDC_NATIVERES, theApp.GetConfig("nativeres", 1));
 	CheckDlgButton(m_hWnd, IDC_ANISOTROPIC, theApp.GetConfig("AnisotropicFiltering", 0));
+	CheckDlgButton(m_hWnd, IDC_ACCURATE_BLEND, theApp.GetConfig("accurate_blend", 0));
```
Comment (gregory38) [inline plugins/GSdx/GSSettingsDlg.cpp]: Please enable by default accurate_blend (replace 0 by 1)

**Comment by gregory38** [comment]
I think you can squash the 3 commits.

**Comment by ssakash** [comment]
I made a mistake, I did the normal push instead of force push so the changes weren't updated. :p 

The changes are done now. :smile:

**Comment by gregory38** [comment]
Was the code compiled and tested?

**Comment by ssakash** [comment]
yes, it works fine.

**Pull request merged**